pkg/models/playlist.go

```diff
@@ -6,19 +6,22 @@ import (
 
 // Typed errors
 var (
-	ErrPlaylistNotFound = errors.New("Playlist not found")
+	ErrPlaylistNotFound                = errors.New("Playlist not found")
+	ErrPlaylistFailedGenerateUniqueUid = errors.New("failed to generate unique playlist UID")
 )
 
 // Playlist model
 type Playlist struct {
 	Id       int64  `json:"id"`
+	Uid      string `json:"uid"`
```

Comment:
I have this minor suggestion so that it conforms with the [go recommendations](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)
```suggestion
	UID      string `json:"uid"`
```